QMLComponents/components/JASP/Controls/ExpanderButton.qml

```diff
@@ -144,6 +140,7 @@ FocusScope
 		anchors.topMargin		: 15 * preferencesModel.uiScale
 		width					: parent.width
 		columns					: 2
+		visible					: expanded //set to that focus chain does not enter closed expanders
```

Comment:
This will make the Section invisible immediately before the animation is completed.
Better to add this at the end of the animation: onStopped: expanderArea.visible = expanded

Reply:
or `visible: expanderWrapper.implicitHeight > 0` because then you just have a binding instead of an imperative.

Reply:
(the animation will make sure the delay is there and if no animation there needs no delay)
Also what happens if animations are disabled....